src/Components/Nav/Components/ProfileDetail.js

```diff
@@ -1,63 +1,48 @@
-import React, { Component } from "react";
-import { Link } from "react-router-dom";
+import React, { useState } from "react";
 import styled from "styled-components";
 import LoginModal from "./LoginModal";
+import SignupModal from "./SignupModal";
 
-export default class ProfileDetail extends Component {
-  constructor() {
-    super();
-    this.state = {
-      isModalActive: false,
-    };
-  }
+export default function ProfileDetail() {
+  const [isLoginActive, setIsLoginActive] = useState(false);
+  const [isSignupActive, setIsSignupActive] = useState(false);
 
-  handleLogin = () => {
-    const { isModalActive } = this.state;
-    this.setState({
-      isModalActive: !isModalActive,
-    });
+  const handleLogin = () => {
```

Comment:
```suggestion
  const toggleLogin = () => {
```
토글한다는 뜻이 들어가면 좋을 것 같습니다.